codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -53,11 +57,14 @@
  */
 public class Http2ConnectionHandler extends ByteToMessageDecoder implements Http2LifecycleManager {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(Http2ConnectionHandler.class);
+    private static final int DEFAULT_GRACEFUL_SHUTDOWN_TIMEOUT_MILLIS = 15 * 60 * 1000; // 15 minutes
```

Comment:
@ejona86 - I would like to be more aggressive as well (previously set to 5 seconds).

@nmittler - Would 1 minute or 30 seconds work for you?